recipe/conda_forge_ci_setup/upload_or_check_non_existence.py

```diff
@@ -42,7 +42,8 @@ def get_built_distribution_names(recipe_dir, variant):
     for m in skipped_distributions:
         print("{} configuration was skipped in build/skip.".format(m.name()))
 
-    return set([m.name() for m, _, _ in metas if not m.skip()])
+    subdirs = set([m.config.target_subdir for m, _, _ in metas if not m.skip()])
+    return set([m.name() for m, _, _ in metas if not m.skip()]), subdirs
```

Comment:
Sets may not be ordered. I think you need lists here.

Reply:
Instead of all combinations of names, we can just use the actual ones built.

Reply:
Why is an ordered list needed?

Reply:
Ah, I see what you are saying. I think that's a clean up for another PR.